backend/.prettierrc

```diff
@@ -3,5 +3,8 @@
   "singleQuote": true,
   "endOfLine": "lf",
   "trailingComma": "none",
-  "printWidth": 120
+  "printWidth": 120,
+  ["error", {
```

Comment:
This is showing me invalid json

Reply:
Removed

> 
> 
> The changes to the EDDN listener journal file is kind of not needed from your end. I was planning to delete it anyway. So, give me a moment while I delete that code and then you can rebase this PR onto the master.

I've deleted this file and checked with a no-commit that it should merge without issues. Ready for re-review